common/controller_runtime_utils.go

```diff
@@ -56,3 +57,12 @@ func GetNodeNameEnvVar() string {
 func GetWatchNameSpaceEnfVar() string {
 	return os.Getenv("WATCH_NAMESPACE")
 }
+
+// GetWatchNamespace returns the namespace the operator should be watching for changes
+func GetWatchNamespace() (string, error) {
```

Comment:
Would it be possible to merge it with `GetWatchNameSpaceEnfVar` just above? I.e. rework `GetWatchNameSpaceEnfVar` to call `GetWatchNamespace()` (and fix the typo in the function name :-)) or even remove it?

Reply:
removed